src/components/Header/Navbar.jsx

```diff
@@ -1,39 +1,34 @@
 import React, { useState } from "react";
 import User from "./User";
 import Language from "./Language";
+import { FaEnvelope, FaBell, FaCaretDown, FaQuestionCircle } from "react-icons/fa";
 
 export default function Navbar() {
-  const [isLanguageDropdownOpen, setIsLanguageDropdownOpen] = useState(false);
   const [isUserDropdownOpen, setIsUserDropdownOpen] = useState(false);
 
-  const toggleLanguageDropdown = () => {
-    setIsLanguageDropdownOpen(!isLanguageDropdownOpen);
-    setIsUserDropdownOpen(false);
-  };
-
   const toggleUserDropdown = () => {
     setIsUserDropdownOpen(!isUserDropdownOpen);
-    setIsLanguageDropdownOpen(false);
   };
 
   return (
-    <>
-      <nav className="font-sans flex flex-col items-center text-center border-b-2 border-green-noQ sm:flex-row sm:text-left sm:justify-between pb-4 px-6 bg-white shadow sm:items-baseline">
-        <div className="my-6 text-3xl sm:mb-0 lg:flex justify-center">
-          Välkommen
+    <nav className="flex items-center justify-between p-4 bg-white shadow-md">
+      <div className="my-6 text-3xl sm:mb-0 lg:flex justify-center font-bold">Stockholm HÃ¤rbarget</div>
```

Comment:
Add functionality to fetch the host name from the backend and use that instead of hard coded value. For example /api/host/count_bookings returns host name (=härberga) for the host that is logged in.

Reply:
fixed